File: PATTERNS/Pattern8/Pattern8.py
# 1
# 2  3
# 4  5  6
# 7  8  9  10
# 11 12 13 14 15

n = int(input("enter the row number:"))
count = 0
for i in range(1,n+1):
    for j in range(1,i+1):
        count=count+1
        print(count," ",end='')
    print()

File: PATTERNS/Pattern08/Pattern8.java
import java.util.Scanner;

// 1
// 2  3
// 4  5  6
// 7  8  9  10
// 11 12 13 14 15

public class Pattern8 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Enter the row number:");
        int n = in.nextInt();
        int count = 0;
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print(++count +" ");
            }
            System.out.println();
        }
    }
}